examples/async-portals/webpack.config.babel.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+const path = require('path')
+const webpack = require('webpack')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  entry: ['./src/index'],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        //exclude: /node_modules|packages/,  // should work without exclude
```

Comment:
Excluding `node_modules` is a good thing, please add it and remove comment.

Reply:
As long CRA will not exclude node_modules and some projects might follow the same approach - we shall be ready (and we are ready).